src/Compilers/CSharp/Test/Syntax/Parsing/PatternParsingTests.cs

```diff
@@ -12701,5 +12727,66 @@ public void ConditionalAsConstantPatternInSwitchStatement_Parenthesized()
             }
             EOF();
         }
+
+        [Fact, WorkItem(53011, "https://github.com/dotnet/roslyn/issues/53011")]
+        public void InvalidPropertyPattern()
+        {
+            UsingExpression(@"new object() is { {}: 1 }", TestOptions.RegularWithPatternCombinators,
```

Comment:
tbh, I didn't understand this parse. It makes it seem like `{ {} }` is a valid pattern, it's just to do `: 1`, we should have inserted a comma in between. What am I missing?

Reply:
the syntax model for this allows a sequence of pattern children, invalid forms of which are reported about if they are syntactically ok.

Reply:
> It makes it seem like { {} } is a valid pattern

during parsing it is. asdasd